exporter/datadogexporter/metrics_exporter.go

```diff
@@ -55,7 +55,10 @@ func translatorFromConfig(set component.TelemetrySettings, cfg *Config, attrsTra
 	options := []otlpmetrics.TranslatorOption{
 		otlpmetrics.WithDeltaTTL(cfg.Metrics.DeltaTTL),
 		otlpmetrics.WithFallbackSourceProvider(sourceProvider),
-		otlpmetrics.WithRemapping(),
+	}
+
+	if cfg.Metrics.RemapMetrics {
```

Comment:
Please log a warning when this is false users will have a broken experience with OOTB dashboards.